public/src/smart_resize.js

```diff
@@ -0,0 +1,46 @@
+import { throttle } from 'lodash';
+
+module.exports = function (editor) {
+  let resize = editor.resize;
+  let lastDemensions = [];
+
+  let checkDelay = 250;
+  let recheckDelay = 500;
+  let stableAfter = 5000;
+  let timeouts = [];
+
+  let stableChecks = 0;
+  let maxStableChecks = stableAfter / recheckDelay;
+
+  function exec() {
+    timeouts.splice(0).map(clearTimeout);
```

Comment:
Yeah, I do `array.splice(0).forEach(...)` all the time, but I get that it's not valuable complexity here. Fixed.